azure-pipeliens.yml

```diff
@@ -2,26 +2,67 @@ jobs:
  - template: default.yml@templates
    parameters:
      minrust: 1.56.0
-     nightly: true
  - template: coverage.yml@templates
    parameters:
      token: $(CODECOV_TOKEN_SECRET)
-     nightly: true
  - job: no_std
    displayName: "Compile-check on no_std target"
    pool:
-     vmImage: ubuntu-16.04
+     vmImage: ubuntu-20.04
    steps:
      - template: install-rust.yml@templates
        parameters:
          targets:
-          - thumbv7m-none-eabi
-     - bash: cargo check --target thumbv7m-none-eabi --no-default-features
+          - aarch64-unknown-none
```

Comment:
Why change this?

Reply:
`thumbv7m-none-eabi` is 32 bit and when I started this PR AtomicU64 was imported unconditionally for `due_time`, so I switched it to `aarch64-unknown-none` mindlessly so that it would compile, however now that `due_time` is behind a #[cfg], checking on both `thumbv7m-none-eabi` and `aarch64-unknown-none` work. I will revert this and add another check for `aarch64-unknown-none` to look for 32/64 bit `no_std` compilation edge cases.